<?xml version="1.0"?>
<launch>

    <!-- UR10e State Machine -->
    <node pkg="iris_cobot" type="ur10e_smach.py" name="ur10e_smach" output="screen" respawn="true"/>

    <!-- State Machine Viewer -->
    <node pkg="smach_viewer" type="smach_viewer.py" name="smach_viewer" output="screen" respawn="true"/>

    <!-- Tap interaction detector -->
    <node pkg="iris_cobot" type="double_tap.py" name="double_tap" output="screen" respawn="true"/>

</launch>